interfaces/console/package.json

```diff
@@ -60,6 +60,7 @@
     "react-xarrows": "^2.0.2",
     "recoil": "^0.7.7",
     "recoil-persist": "^4.2.0",
+    "sharp": "^0.32.6",
```

Comment:
What is this lib for ?

Reply:
Next. js built-in image optimization requires sharp as a dependency
https://www.npmjs.com/package/sharp